components/idp-mgt/org.wso2.carbon.saml.metadata/pom.xml

```diff
@@ -0,0 +1,151 @@
+                            org.wso2.carbon.registry.core.exceptions,
+                            org.wso2.carbon.registry.core.jdbc.utils,
+                            org.wso2.carbon.registry.core.service,
+                            org.wso2.carbon.registry.core.session,
+                            org.apache.axiom.*; version="${axiom.osgi.version.range}",
+                            org.apache.axis2.*; version="${axis2.osgi.version.range}",
+                            javax.xml.stream,
+                            org.wso2.carbon.utils.multitenancy,
+                            org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
+                            org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}",
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.core; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.utils; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.*; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.core.util; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.base; version="${carbon.base.imp.pkg.version.range}",
+                            org.wso2.carbon.stratos.common.*; version="${carbon.commons.imp.pkg.version}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.common.*;version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.core.*;version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.base;version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.core.persistence;version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon;version="${carbon.kernel.package.import.version.range}",
+                            org.apache.commons.codec.binary; version="${commons-codec.wso2.osgi.version.range}",
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.saml.metadata.internal,
+                            org.wso2.carbon.saml.metadata.*; version="${carbon.identity.package.export.version}",
```

Comment:
carbon.identity.package.export.version doesn't sound right as we no longer have carbon-identity component.
shall we rename this property to carbon.identity.framework.package.export.version